Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:6
+
+# Screwdriver Store Version
+ARG VERSION=latest
```

Comment:
Is this the store version or the queue-worker version or both? I see `$VERSION` used below in the `npm install` step